packages/pages/etc/pages.api.md

```diff
@@ -112,6 +112,7 @@ export interface ModuleClientServerRenderTemplates {
 
 // @public
 export interface ModuleConfig {
+  domain?: string;
```

Comment:
Can you move this after name?

Reply:
odd, it is in the types.ts file but not in that documentation.